query_optimizer/tests/execution_generator/Partition.test

```diff
@@ -15,19 +15,35 @@
 # specific language governing permissions and limitations
 # under the License.
 
-CREATE TABLE foo (id INT NULL,
-                  name CHAR(20))
+CREATE TABLE dim (id INT NULL,
```

Comment:
Optional, but it may be easier to read if the relation is named as ``dim_4_hash_partitions``.